.github/ISSUE_TEMPLATE/tech-debt.md

```diff
@@ -2,7 +2,7 @@
 name: Technical Debt
 about: Report technical debt to improve our code
 title: ''
-labels: 'technical debt'
+labels: technical debt
```

Comment:
Doesn't this mean that it will get two labels `technical` and `debt`?

Reply:
Funnily enough, I didn't change this, GH must have done this for some reason. It could be a bug. We can drop this from the PR to be safe.